src/test/java/uk/gov/hmcts/sptribs/common/event/page/ContactPreferenceDetailsTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.sptribs.common.event.page;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.sptribs.ciccase.model.CaseData;
+import uk.gov.hmcts.sptribs.ciccase.model.CaseSubcategory;
+import uk.gov.hmcts.sptribs.ciccase.model.CicCase;
+import uk.gov.hmcts.sptribs.ciccase.model.PartiesCIC;
+import uk.gov.hmcts.sptribs.ciccase.model.State;
+import uk.gov.hmcts.sptribs.ciccase.model.SubjectCIC;
+
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+public class ContactPreferenceDetailsTest {
+
+    @InjectMocks
+    private ContactPreferenceDetails contactPreferenceDetails;
+
+    @Test
+    void shouldSelectContactPreference() {
+        //Given
+        final CaseDetails<CaseData, State> caseDetails = new CaseDetails<>();
+        final CicCase cicCase = CicCase.builder()
+            .caseSubcategory(CaseSubcategory.SEXUAL_ABUSE)
+            .partiesCIC(Set.of(PartiesCIC.APPLICANT))
+            .build();
+        final CaseData caseData = CaseData.builder()
+            .cicCase(cicCase)
+            .build();
+        caseDetails.setData(caseData);
+
+        //When
+        final AboutToStartOrSubmitResponse<CaseData, State> response = contactPreferenceDetails.midEvent(caseDetails, caseDetails);
```

Comment:
For consistency in the code, what is the preference with `final` modifiers?
Here on line 40 the response is `final`, but in the `CreateCaseTest`, several responses are not marked as final.